metaflow/runner/deployer_impl.py

```diff
@@ -63,11 +63,20 @@ def __init__(
 
         from metaflow.parameters import flow_context
 
-        if "metaflow.cli" in sys.modules:
-            # Reload the CLI with an "empty" flow -- this will remove any configuration
-            # options. They are re-added in from_cli (called below).
-            with flow_context(None) as _:
-                importlib.reload(sys.modules["metaflow.cli"])
+        # Reload the CLI with an "empty" flow -- this will remove any configuration
```

Comment:
Few nits:
 - change comment to say that it is configuration *AND* parameter options
 - it is `init_cmd` (no s). The other one is correct.